### ISSUE TITLE
Secure Video?

### ISSUE THREAD

**Issue by robertgranholm** [hjdhjd/homebridge-unifi-protect#282]
**Is your feature request related to a problem? Please describe:**

NO

**Describe the solution you'd like:**

I'd like to see, or know why "Secure Video" implemented.

Is there any consideration or way to implement our cameras being considered "Secure Video" by Apple Homekit, so that we can reap the benefits of activity zones and other "Secure Video" only features?

I was going back and forth with a friend about how this implementation I have with Unifi is the best, and he kept harping on the use of activity zones, and thumbnail notifications...and I wanted to set that up, ran into the fact that it's not supported outside of Apple's Secure Video feature set.

**Describe alternatives you've considered:**

None, unless you count not using Unifi.

**Additional context:**

**Comment by hjdhjd** [comment]
Asked and answered. Please search prior reports. :smile:

And thumbnail notifications have nothing to do with HKSV, and in fact are supported.  😉 

TL;DR: in time. This is on the roadmap...I will make zero commitments on timing.
